public/examples/card-on-file-payment-with-verification.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="/app.css" rel="stylesheet" />
+    <script
+      type="text/javascript"
+      src="https://sandbox.web.squarecdn.com/v1/square.js"
+    ></script>
+    <script>
+      const appId = '{APPLICATION_ID}';
+      const locationId = '{LOCATION_ID}';
+
+      async function createCardOnFilePayment(
+        sourceId,
+        customerId,
+        verificationToken
+      ) {
+        const bodyParameters = {
+          locationId,
+          sourceId,
+          customerId,
+          verificationToken,
+        };
+
+        const body = JSON.stringify(bodyParameters);
+
+        const paymentResponse = await fetch('/payment', {
```

Comment:
Payment does not provide a synchronous endpoint ?

Reply:
fetch returns a Promise so if you want to have sync processing you need to await